### PR TITLE
Enable various tests on Mac OS

### PR DESCRIPTION
**Pull request by snecklifter**

### CODE REVIEW

File: spec/lilypond_spec.rb [hunk 1]
```diff
@@ -4,10 +4,10 @@
 \\relative c' { f d f a d f e d cis a cis e a g f e }
 eos
 
-describe Asciidoctor::Diagram::LilypondBlockMacroProcessor, :broken_on_windows, :broken_on_osx do
+describe Asciidoctor::Diagram::LilypondBlockMacroProcessor, :broken_on_osx, :broken_on_windows do
```
Comment (pepijnve) [inline spec/lilypond_spec.rb:7]: I think you want to remove `broken_on_osx` here

Comment (snecklifter) [reply spec/lilypond_spec.rb:7]: @pepijnve lilypond is still a work in progress atm.

Comment (snecklifter) [reply spec/lilypond_spec.rb:7]: I can remove the section from the workflow or comment it out while I work on it.

Comment (pepijnve) [reply spec/lilypond_spec.rb:7]: Ok, no problem. Lilypond is a bit of a pain on macOS at the moment indeed due to the lack of 64-bit binaries.

Comment (snecklifter) [reply spec/lilypond_spec.rb:7]: I wont let it defeat me @pepijnve !

Comment (pepijnve) [reply spec/lilypond_spec.rb:7]: That's the spirit! 💪

**Pull request merged**